### PR TITLE
[SendGrid Lists Destination] - doc for new SendGrid Lists Destination

### PR DESCRIPTION
**Pull request by joe-ayoub-segment**
### Proposed changes

Adding a new doc for SendGrid Lists Destination. 
This is an Audience / Engage Destination. 

### Merge timing

Please do not deploy this doc live until we've moved the Destination into Public Beta. It is in Private Beta at the moment and is being tested. We'll notify you on this ticket when we'd like the doc published live. 

### Related issues (optional)

N/A

### CODE REVIEW

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### Does Segment create new Contacts in SendGrid?
+Segment will create Contacts in SendGrid if a Contact doesn't already exist.
+
+### Will Segment delete Contacts from SendGrid?
+Segment will not delete Contacts from SendGrid. If a user is removed from an Engage Audience, Segment will remove the Contact from the associated List in SendGrid but will not delete the Contact. 
+
+### Sending additional user traits
+Segment supports sending Engage user profile traits to SendGrid Contact User Attributes. The following additional manual configuration steps are required: 
+
+1. Use Engage's [Trait Enrichment](https://segment.com/docs/engage/trait-activation/trait-enrichment/) to configure Engage to include specific user profile traits when syncing users to the SendGrid List.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:57]:
```suggestion
1. Use [Trait Enrichment](/docs/engage/trait-activation/trait-enrichment/) to include specific user profile traits when syncing users to a SendGrid List. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Additional information
+
+### Does Segment create Lists in SendGrid?
+Segment automatically creates Lists in SendGrid. The List will be named with the value provided in the **Name** field. If no name is provided in the **Name** field Segment will default to using the Audience's **Audience Key** value. 
+
+### Does Segment create new Contacts in SendGrid?
+Segment will create Contacts in SendGrid if a Contact doesn't already exist.
+
+### Will Segment delete Contacts from SendGrid?
+Segment will not delete Contacts from SendGrid. If a user is removed from an Engage Audience, Segment will remove the Contact from the associated List in SendGrid but will not delete the Contact.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:52]:
```suggestion
Segment doesn't delete Contacts from SendGrid. If you remove a user from an Engage Audience, Segment does remove the Contact from the associated SendGrid List, but doesn't delete the Contact from SendGrid. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+1. From your Segment workspace's home page, click **Connections** > **Destinations**, then search for the **Destination** created in previous steps and click on its tile.
+2. Click on **Mappings** > **New Mapping** > **Sync Audience** > **Save**. 
+3. On the next screen, enable the Mapping using the **Status** toggle.
+
+### Connect an Audience
+
+1. From your Segment workspace's home page, navigate to the Audience you'd like to sync to SendGrid by clicking on **Engage** > **Audiences**, then selecting the Audience to connect. 
+2. Click **Add Destination**, then search for the **SendGrid Lists (Actions)** instance you created earlier and select its tile. 
+3. Enter a **List Name**, select **Default Setup**, and click **Save**. On the following screen, click **Add 1 Destination**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:36]:
```suggestion
3. Enter a **List Name**, select **Default Setup**, and click **Save**. When prompted, select **Add 1 Destination**. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+### Create a Mapping
+
+1. From your Segment workspace's home page, click **Connections** > **Destinations**, then search for the **Destination** created in previous steps and click on its tile.
+2. Click on **Mappings** > **New Mapping** > **Sync Audience** > **Save**. 
+3. On the next screen, enable the Mapping using the **Status** toggle.
+
+### Connect an Audience
+
+1. From your Segment workspace's home page, navigate to the Audience you'd like to sync to SendGrid by clicking on **Engage** > **Audiences**, then selecting the Audience to connect. 
+2. Click **Add Destination**, then search for the **SendGrid Lists (Actions)** instance you created earlier and select its tile.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:35]:
```suggestion
2. Click **Add Destination**, and select the SendGrid Lists (Actions) destination you previously created.
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### Create a Mapping
+
+1. From your Segment workspace's home page, click **Connections** > **Destinations**, then search for the **Destination** created in previous steps and click on its tile.
+2. Click on **Mappings** > **New Mapping** > **Sync Audience** > **Save**. 
+3. On the next screen, enable the Mapping using the **Status** toggle.
+
+### Connect an Audience
+
+1. From your Segment workspace's home page, navigate to the Audience you'd like to sync to SendGrid by clicking on **Engage** > **Audiences**, then selecting the Audience to connect.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:34]:
```suggestion
1. From your Segment workspace's home page, navigate to **Engage** > **Audiences** and select the Audience you'd like to sync to SendGrid. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+### Create and connect 'SendGrid Lists (Actions)' to an Engage Space
+
+1. From your Segment workspace's home page, click on **Engage** > **Engage Settings** > **Settings**. Click the **Add destination** button. 
+2. Search for **SendGrid Lists (Actions)** and select its tile. Click **Add Destination** and then **Confirm Source**.
+3. On the Basic Settings screen provide **Name** and **API Key** values in the specified fields. Enable the Destination. 
+
+### Create a Mapping
+
+1. From your Segment workspace's home page, click **Connections** > **Destinations**, then search for the **Destination** created in previous steps and click on its tile.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:28]:
```suggestion
1. From your Segment workspace's home page, click **Connections** > **Destinations** and select the SendGrid Lists (Actions) destination you previously created.
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected. 
+3. Save the API key value securely, as you will need it in later steps.
+
+### Create and connect 'SendGrid Lists (Actions)' to an Engage Space
+
+1. From your Segment workspace's home page, click on **Engage** > **Engage Settings** > **Settings**. Click the **Add destination** button. 
+2. Search for **SendGrid Lists (Actions)** and select its tile. Click **Add Destination** and then **Confirm Source**.
+3. On the Basic Settings screen provide **Name** and **API Key** values in the specified fields. Enable the Destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:24]:
```suggestion
3. On the Basic Settings screen, provide **Name** and **API Key** values in the specified fields, toggle "Enable destination" to on, and then click **Save Changes**. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected. 
+3. Save the API key value securely, as you will need it in later steps.
+
+### Create and connect 'SendGrid Lists (Actions)' to an Engage Space
+
+1. From your Segment workspace's home page, click on **Engage** > **Engage Settings** > **Settings**. Click the **Add destination** button. 
+2. Search for **SendGrid Lists (Actions)** and select its tile. Click **Add Destination** and then **Confirm Source**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:23]:
```suggestion
2. Search for SendGrid Lists (Actions) and select the SendGrid Lists (Action) tile. Click **Add Destination** and **Confirm Source**.
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+**SendGrid Lists (Actions)** is designed to work with **Engage Audiences** only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. **SendGrid Lists (Actions)** is not designed to connect to regular Connections Sources. 
+
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected. 
+3. Save the API key value securely, as you will need it in later steps.
+
+### Create and connect 'SendGrid Lists (Actions)' to an Engage Space
+
+1. From your Segment workspace's home page, click on **Engage** > **Engage Settings** > **Settings**. Click the **Add destination** button.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:22]:
```suggestion
1. From your Segment workspace's home page, navigate to **Engage** > **Engage Settings** > **Settings** and click **Add destination**. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+## Getting started
+
+**SendGrid Lists (Actions)** is designed to work with **Engage Audiences** only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. **SendGrid Lists (Actions)** is not designed to connect to regular Connections Sources. 
+
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected. 
+3. Save the API key value securely, as you will need it in later steps.
+
+### Create and connect 'SendGrid Lists (Actions)' to an Engage Space
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:20]:
```suggestion
### Connect a SendGrid Lists (Actions) destination to an Engage Space
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+This destination is maintained by Segment. For any issues with the destination [contact the Segment Support team](mailto:friends@segment.com).
+
+## Getting started
+
+**SendGrid Lists (Actions)** is designed to work with **Engage Audiences** only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. **SendGrid Lists (Actions)** is not designed to connect to regular Connections Sources. 
+
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected. 
+3. Save the API key value securely, as you will need it in later steps.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:18]:
```suggestion
3. Save the API key value in a secure location, as you will need it in later steps.
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+This destination is maintained by Segment. For any issues with the destination [contact the Segment Support team](mailto:friends@segment.com).
+
+## Getting started
+
+**SendGrid Lists (Actions)** is designed to work with **Engage Audiences** only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. **SendGrid Lists (Actions)** is not designed to connect to regular Connections Sources. 
+
+### Create a SendGrid API Key
+1. Sign in to your SendGrid account, then navigate to **Settings** > **API Keys**.
+2. Click **Create API Key** and follow the instructions to generate a new API key. Ensure Full Access permission is selected.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:17]:
```suggestion
2. Click **Create API Key** and follow the instructions to generate a new API key. Be sure to grant the API key **Full Access** permission. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+{% include content/plan-grid.md name="actions" %}
+
+[SendGrid Lists (Actions)](https://mc.sendgrid.com/contacts/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} help customers organize their email recipients into targeted groups, enabling them to send personalized, relevant content to specific audiences. This improves engagement, increases email deliverability, and streamlines campaign management.
+
+This destination is maintained by Segment. For any issues with the destination [contact the Segment Support team](mailto:friends@segment.com).
+
+## Getting started
+
+**SendGrid Lists (Actions)** is designed to work with **Engage Audiences** only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. **SendGrid Lists (Actions)** is not designed to connect to regular Connections Sources.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:13]:
```suggestion
SendGrid Lists (Actions) is designed to work with Engage Audiences only. The steps below outline how to create and connect the Destination to Engage and then to an Engage Audience. SendGrid Lists (Actions) is not designed to connect to Connections Sources. 
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
+---
+title: [SendGrid Lists (Actions)] Destination
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[SendGrid Lists (Actions)](https://mc.sendgrid.com/contacts/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} help customers organize their email recipients into targeted groups, enabling them to send personalized, relevant content to specific audiences. This improves engagement, increases email deliverability, and streamlines campaign management.
+
+This destination is maintained by Segment. For any issues with the destination [contact the Segment Support team](mailto:friends@segment.com).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:9]:
```suggestion
This destination is maintained by Segment. For any issues with the destination, [contact the Segment Support team](mailto:friends@segment.com).
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
+---
+title: [SendGrid Lists (Actions)] Destination
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[SendGrid Lists (Actions)](https://mc.sendgrid.com/contacts/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} help customers organize their email recipients into targeted groups, enabling them to send personalized, relevant content to specific audiences. This improves engagement, increases email deliverability, and streamlines campaign management.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:7]:
```suggestion
[SendGrid Lists (Actions)](https://mc.sendgrid.com/contacts/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps customers organize their email recipients into targeted groups, enabling them to send personalized, relevant content to specific audiences. This improves engagement, increases email deliverability, and streamlines campaign management.
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
+---
+title: [SendGrid Lists (Actions)] Destination
+---
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:3]: The `engage: true` attribute adds the "This destination is only compatible with [Twilio Engage](https://segment.com/docs/engage)." to the destination dossier. For example, https://segment.com/docs/connections/destinations/catalog/actions-display-video-360/

Please update the `id` attribute when you have it. 

```suggestion
---
title: SendGrid Lists (Actions) Destination
engage: true
id: 
---
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+1. From your Segment workspace's home page, navigate to the Audience you'd like to sync to SendGrid by clicking on **Engage** > **Audiences**, then selecting the Audience to connect. 
+2. Click **Add Destination**, then search for the **SendGrid Lists (Actions)** instance you created earlier and select its tile. 
+3. Enter a **List Name**, select **Default Setup**, and click **Save**. On the following screen, click **Add 1 Destination**. 
+
+The **SendGrid Lists (Actions)** will now start to sync the **Engage Audience** to a **SendGrid List**.
+
+{% include components/actions-fields.html %}
+
+
+## Additional information
```
[the remark this answers is not in the document]
Comment (joe-ayoub-segment) [reply src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:45]: hi @forstisabella yes absolutely! Do you want to do it or shall I?

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+1. From your Segment workspace's home page, navigate to **Engage** > **Audiences** and select the Audience you'd like to sync to SendGrid. 
+2. Click **Add Destination**, and select the SendGrid Lists (Actions) destination you previously created.
+3. Enter a **List Name**, select **Default Setup**, and click **Save**. When prompted, select **Add 1 Destination**. 
+
+The SendGrid Lists (Actions) destination will now start to sync your Engage Audience to a SendGrid List.
+
+{% include components/actions-fields.html %}
+
+
+## Additional information
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:45]:
```suggestion
## Troubleshooting
```

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+### Does Segment create Lists in SendGrid?
+Segment automatically creates Lists in SendGrid. If you provide a value in the **Name** field, Segment names the List the value you provided. If you do not provide a name in the  **Name** field, Segment gives the List the Engage Audience's **Audience Key** value. 
+
+### Does Segment create new Contacts in SendGrid?
+Segment creates Contacts in SendGrid if a Contact doesn't already exist for the user.
+
+### Does Segment delete Contacts from SendGrid?
+Segment doesn't delete Contacts from SendGrid. If you remove a user from an Engage Audience, Segment does remove the Contact from the associated SendGrid List, but doesn't delete the Contact from SendGrid. 
+
+### Sending additional user traits
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:56]:
```suggestion
## Best practices

### Sending additional user traits
```

**Comment by tcgilbert** [comment]
@joe-ayoub-segment you can also add the hidden frontmatter to this if you want to go ahead and merge but keep it hidden: 

```
---
title: SendGrid Lists (Actions) Destination
engage: true
id: 67338e95bf70aed334093dae
hidden: true
```

**Comment by joe-ayoub-segment** [comment]
hi @forstisabella can we get this live now please? We wait it hidden though.

File: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md [hunk 20]
```diff
@@ -0,0 +1,85 @@
...
+## Best practices
+
+### Sending additional user traits
+Segment supports sending Engage user profile traits to SendGrid Contact User Attributes. The following additional manual configuration steps are required: 
+
+1. Use [Trait Enrichment](/docs/engage/trait-activation/trait-enrichment/) to include specific user profile traits when syncing users to a SendGrid List. 
+2. Standard User Attributes: Use the [Sync Audience Action](#sync-audience-action)'s User Attributes field to map the following [Contact Profile Fields](https://www.twilio.com/docs/sendgrid/ui/managing-contacts/segmenting-your-contacts#contact-profile-fields){:target="_blank”} to SendGrid:
+ - First Name
+ - Last Name
+ - Phone Number (must be in [E.164](https://www.twilio.com/docs/glossary/what-e164) format)
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-sendgrid-audiences/index.md:66]:
```suggestion
 - Phone Number (must be in [E.164](https://www.twilio.com/docs/glossary/what-e164){:target="_blank”} format)
```

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
We're waiting on a review from @tcgilbert and then we'll be g2g!

> hi @forstisabella can we get this live now please? We wait it hidden though.

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**